Assets/Mirror/Core/NetworkManagerHUD.cs

```diff
@@ -23,25 +23,16 @@ void OnGUI()
         {
             GUILayout.BeginArea(new Rect(10 + offsetX, 40 + offsetY, 300, 9999));
             if (!NetworkClient.isConnected && !NetworkServer.active)
-            {
                 StartButtons();
-            }
             else
-            {
                 StatusLabels();
-            }
 
             // client ready
-            if (NetworkClient.isConnected && !NetworkClient.ready)
```

Comment:
please keep the guilayout part on a separate line, that's much easier to read

Reply:
This PR will be redone anyway - was just for Jesus to look at.